english/disadvantage_of_git

```diff
@@ -0,0 +1 @@
+Subproject commit 1018a704d5a7b8c6c202006be542f3feff2264a7
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.